corehq/apps/commtrack/static/commtrack/js/sms.js

```diff
@@ -184,15 +184,21 @@ hqDefine('commtrack/js/sms.js', function () {
             };
         };
     }
-    return {
-        initCommtrackSettingsView: function ($element, settings, other_sms_codes) {
-            var model = new CommtrackSettingsViewModel(other_sms_codes);
-            $element.submit(function () {
-                return model.presubmit();
-            });
 
-            model.load(settings);
-            $element.koApplyBindings(model);
-        }
-    };
+    function initCommtrackSettingsView($element, settings, other_sms_codes) {
+        var model = new CommtrackSettingsViewModel(other_sms_codes);
+        $element.submit(function () {
+            return model.presubmit();
+        });
+
+        model.load(settings);
+        $element.koApplyBindings(model);
+    }
+
+    $(function () {
```

Comment:
This document ready handler is in a `hqDefine` block while many in this refactoring effort [are](https://github.com/dimagi/commcare-hq/pull/15117/files#diff-db25bc0724e2064bf48e3997942a271a) [not](https://github.com/dimagi/commcare-hq/pull/15117/files#diff-0488a0e5bbab1d5d0a4c695201d52120). Do you have any thoughts on which is better, esp. for best practice recommendations?